### PR TITLE
fix compile error with bindgen 0.58

### PR DESCRIPTION
**Pull request by aurabindo**
Signed-off-by: Aurabindo Pillai <mail@aurabindo.in>

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -32,7 +32,6 @@ fn main() {
     let bindings = include_args
         .fold(bindings, |bindings, arg| bindings.clang_arg(arg))
         .header("src/wrapper.h")
-        .link("drm")
```
Comment (devurandom) [inline build.rs:35]: Why is this no longer necessary?

Comment (aurabindo) [reply build.rs:35]: The option was removed https://github.com/rust-lang/rust-bindgen/commit/8e3e585b6efc6771cac8a9fa6ef9f9310c3d0f08 
Not sure if additional changes are necessary but the comment says it wasnt really doing anything

Comment (devurandom) [reply build.rs:35]: How do we now make sure to load `libdrm.so.123` at runtime? (Sorry, my Rust is rusty, haven't been looking at this code in a while.)

Comment (aurabindo) [reply build.rs:35]: Just did some digging and looks like we could use the link attribute, but also `println!("cargo:rustc-link-lib=drm")` should also work. Do you have any examples to test it against if the println works fine ?

**Review by devurandom — APPROVED** [review]

**Pull request merged**